modules/embedding-microshift-rpm-ostree-image.adoc

```diff
@@ -0,0 +1,15 @@
+= Prerequisites
+
+You have read the documentation on how to compose RHEL for Edge rpm-ostree imageueprints and build them either on your own RHEL 8 Image Builder machine or on Red Hat Hosted Image Builder service:
```

Comment:
Is "imagueprints" a typo? "image blueprints"?

Reply:
Thanks for catching this. Fixed.